plugins/nominal-connection-checker/test/connection_checker_test_helper.mocha.js

```diff
@@ -199,58 +198,59 @@ export function clearTwoBlockTests() {
  * called at the end of a suite of two block tests.
  */
 export function runTwoBlockTests() {
+  /**
+   * Creates a function which returns the input of an outer block.
+   * @param {string} suffix The suffix for the block type.
+   * @return {function(string, string=): !Blockly.Connection} A function that
+   *     takes in a type name and returns an input connection.
+   */
+  function createGetOuterInput(suffix) {
+    return function(type, name) {
+      const block = this.workspace.newBlock('static_' + type + suffix);
+      const inConn = block.nextConnection ||
+          block.getInput('INPUT1').connection;
+      inConn.name = name || type + 'In';
+      return inConn;
+    };
+  }
+
+  /**
+   * Creates a function which returns the output of an inner block.
```

Comment:
Creates a function which creates a new block and returns its output connection or previous connection.